apps/web-giddh/src/app/theme/new-confirm-modal/confirm-modal.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+    selector: 'new-confirm-modal',
+    templateUrl: './confirm-modal.component.html',
+    styleUrls: ['./confirm-modal.component.scss']
+})
+
```

Comment:
Avoid empty line between `@Component` decorator and class declaration

Reply:
Done